labgrid/remote/exporter.py

```diff
@@ -381,13 +381,9 @@ def onChallenge(self, challenge):
         self.loop.stop()
 
     async def acquire(self, group_name, resource_name):
-        resource = self.groups[group_name][resource_name]
```

Comment:
This (and the one below) should actually be TODOs. We want to be able to perform local actions when a resource is acquired, and this would be the correct place when we get around to that.